server/jetstream_sourcing_scaling_test.go

```diff
@@ -16,12 +16,95 @@ package server
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/nats-io/nats.go"
+	"strconv"
 	"testing"
 	"time"
-
-	"github.com/nats-io/nats.go"
 )
 
+var serverConfig1 = `
+server_name: server1
+listen: 127.0.0.1:4222
+http: 8222
+
+prof_port = 18222
+
+jetstream: {max_mem_store: 256MB, max_file_store: 2GB, store_dir: '%s'}
+
+cluster {
+  name: my_cluster
+  listen: 127.0.0.1:4248
+  routes: [nats://127.0.0.1:4249,nats://127.0.0.1:4250]
+}
+
+accounts { $SYS { users = [ { user: "admin", pass: "s3cr3t!" } ] }}
+	`
+
+var serverConfig2 = `
+server_name: server2
+listen: 127.0.0.1:5222
+http: 8223
+
+prof_port = 18223
+
+jetstream: {max_mem_store: 256MB, max_file_store: 2GB, store_dir: '%s'}
+
+cluster {
+  name: my_cluster
+  listen: 127.0.0.1:4249
+  routes: [nats://127.0.0.1:4248,nats://127.0.0.1:4250]
+}
+
+accounts { $SYS { users = [ { user: "admin", pass: "s3cr3t!" } ] }}
+`
+
+var serverConfig3 = `
+server_name: server3
+listen: 127.0.0.1:6222
+http: 8224
+
+prof_port = 18224
+
+jetstream: {max_mem_store: 256MB, max_file_store: 2GB, store_dir: '%s'}
+
+cluster {
+  name: my_cluster
+  listen: 127.0.0.1:4250
+  routes: [nats://127.0.0.1:4248,nats://127.0.0.1:4249]
+}
+
+accounts { $SYS { users = [ { user: "admin", pass: "s3cr3t!" } ] }}
+`
+
+var connectURL = "nats://127.0.0.1:4222,nats://127.0.0.1:4223,nats://127.0.0.1:4224"
+
+func createMyLocalCluster(t *testing.T) *cluster {
```

Comment:
you can use this helper instead: 
https://github.com/nats-io/nats-server/blob/main/server/jetstream_cluster_4_test.go#L1435-L1457

Reply:
I did it this way rather than using the helper because with the helper I do not control which ports the servers will actually use, and since this is a long running test where I often will monitor things while the test is running (outside of the test itself) it is convenient if the servers' ports are the same each time (so that I have a context setup that I can use to interact directly with the servers from outside the test while the test is running). 

Doesn't affect the test, just a convenience way to easily look at or interact with the servers while the test is running.